framework/widgets/ActiveField.php

```diff
@@ -143,6 +143,12 @@ class ActiveField extends Component
      */
     public $parts = [];
 
+    /**
+     * @var string this property holds a custom input id if it was set using [[inputOptions]] or in one of the
+     * `$options` parameters of the `input*` methods.
+     */
+    private $_inputId;
```

Comment:
I can not think of a case where you would need to override them and if so, you could just override both.